### PR TITLE
Add description to README

### PR DESCRIPTION
**Pull request by JamesLee-Jones**
Add a description of the aim of the project to the README file.

Fixes: #8.

### CODE REVIEW

**Comment by JamesLee-Jones** [comment]
Any suggestions on how this can be extended/improved would be appreciated!

File: README.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
 # nlp-timeline-gen
```
Comment (zeegtab) [inline README.md:1]: Maybe change the title to the name of our project (Automatic Character Timeline Generation from Books (NLP))

File: README.md [hunk 2]
```diff
@@ -1,2 +1,6 @@
...
-A tool for generating character timelines from books using nlp.
+
+Novels often contain many places, characters and organisations each with their own unique names. As the plot thickens,
+these can become difficult to follow.
+nlp-timeline-gen aims to make this easier by generating a timeline of where people are at what times using 
+[state-of-the-art natural language processing](https://spacy.io/).
```
Comment (zeegtab) [inline README.md:6]: You could add a line below (maybe with a divider before) mentioning that this is for the Year 3 SWE Group Project at Imperial College London.

**Review by AbdulRJacob — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (JamesLee-Jones) [reply README.md:6]: Done.

**Pull request merged**